tests/phpunit/Unit/ApiServerUnitTest.php

```
@@ -0,0 +1,101 @@
+      ],
+    ];
+  }
+
+  /**
+   * Test that sending a response prints its body.
+   */
+  public function testSendResponsePrintsBody(): void {
+    $_SERVER['SERVER_PROTOCOL'] = 'HTTP/1.1';
+
+    ob_start();
+    ApiServer::sendResponse(new Response(200, 'OK', ['X-Custom' => 'value'], 'hello'));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\BehatPhpServer\ApiServer\ApiServer' in method 'testSendResponsePrintsBody'. (undefined)

(StaticAccess)